src/components/PropertyValuePair/stories/PropertyName.stories.tsx

```diff
@@ -16,3 +16,19 @@ export const Default = Template.bind({});
 Default.args = {
     children: loremIpsum({ p: 1, avgSentencesPerParagraph: 1, avgWordsPerSentence: 4, random: false }).toString(),
 };
+
+export const EmphasisLabel = Template.bind({});
```

Comment:
We can remove this story because this is already possible to test on the Label component story directly.